net-wireless/crda/crda-3.18-r3.ebuild

```diff
@@ -25,8 +25,8 @@ RDEPEND="!gcrypt? (
 	net-wireless/wireless-regdb"
```

Comment:
both openssl and libressl need `:0=`

Reply:
and also add it to libgcrypt

Reply:
Thanks. Fixed.
I also found out that I cannot bump EAPI, since `has_version: option --host-root is not supported with EAPI 7`, so I reverted back to 6.